airbyte-integrations/connectors/source-snowflake/source_snowflake/streams/table_stream.py

```diff
@@ -249,8 +251,19 @@ def read_records(
         for record in super().read_records(sync_mode, cursor_field, stream_slice, stream_state):
             if isinstance(self.cursor_field, str):
```

Comment:
Do we expect the `cursor_field` could be different from the `str` type? Are there multiple types ?

Reply:
I initialize the cursor as empty list in order to detect if cursor is set by the user or not. if it is set, i update the state, otherwise, I don't update the state value

Reply:
Nice!